app/code/Magento/Deploy/Model/Filesystem.php

```diff
@@ -149,13 +149,13 @@ public function __construct(
      * Regenerate static
      *
      * @param OutputInterface $output
+     * @param bool $resetStaticVersion
      * @return void
      * @throws LocalizedException
      * @throws \Exception
      */
-    public function regenerateStatic(
-        OutputInterface $output
-    ) {
+    public function regenerateStatic(OutputInterface $output, $resetStaticVersion = false)
```

Comment:
Unfortunately, we are not allowed to change public method signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)